### DISCUSSION TITLE
Examples in API docs when not using a pydantic model

### DISCUSSION THREAD

**Discussion by hellocoldworld** [fastapi/fastapi#6929] [Questions] [answered]
Hello, my question is, can I declare examplesI want to appear in the documentation when I'm not using a pydantic model?
For example, when using pydantic, the docs syas we can do it with 

```python
    class Config:
        schema_extra = {
            "example": {
                "name": "Foo",
                "description": "A very nice Item",
                "price": 35.4,
                "tax": 3.2,
            }
        }
 ```
 
Problem is: I have an endpoint that looks something like this:

```python
@app.post("", response_model=Dict[str. Any], status_code=200)
async def evaluate(parameters: Dict[str, Any]) -> Dict[str, Any]:
     ...
```

Even though I'll except every json as valid, I still want to include some examples.
Is there any way I can accomplish this?

* [ :heavy_check_mark: ] I added a very descriptive title to this issue.
* [:heavy_check_mark: ] I used the GitHub search to find a similar issue and didn't find it.
* [:heavy_check_mark: ] I searched the FastAPI documentation, with the integrated search.
* [ :heavy_check_mark:] I already searched in Google "How to X in FastAPI" and didn't find any information.
* [ :heavy_check_mark:] I already read and followed all the tutorial in the docs and didn't find an answer.
* [ :heavy_check_mark:] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
* [:heavy_check_mark: ] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
* [:heavy_check_mark: ] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).
* [ :heavy_check_mark:] After submitting this, I commit to one of:
    * Read open issues with questions until I find 2 issues where I can help someone and add a comment to help there.
    * I already hit the "watch" button in this repository to receive notifications and I commit to help at least 2 people that ask questions in the future.
    * Implement a Pull Request for a confirmed bug.

**Comment by ycd** [comment]
You use `responses` parameter.

```python

@app.post("",
    responses={
        200: {
            "content": {
                "application/json": {
                    "example": {
                        "name": "Foo",
                        "description": "A very nice Item",
                        "price": 35.4,
                        "tax": 3.2,
                   }
                }
            },
        },
    },
)
async def evaluate(parameters: Dict[str, Any]) -> Dict[str, Any]:
     ...

**Comment by Kludex** [comment, marked as answer]
I feel like the question wants this approach instead:

```python
from fastapi import FastAPI, Body
from typing import Dict, Any

app = FastAPI()

@app.post("/", response_model=Dict[str, Any], status_code=200)
async def evaluate(parameters: Dict[str, Any] = Body(..., example={"name": "potato"})) -> Dict[str, Any]:
     ...
``` 

By the way, I don't think this is documented.

**Comment by hellocoldworld** [comment]
Very useful, both! Thank you so much.

**Comment by tiangolo** [comment]
Thanks for the help here everyone! :clap: :bow: 

Thanks for reporting back and closing the issue :+1:
